app/views/volunteers/edit.html.erb

```diff
@@ -5,12 +5,10 @@
     <h1>Editing Volunteer</h1>
     <% if current_user.supervisor? ||
         current_user.casa_admin? %>
-      <%= link_to t(".send_reminder"),
-                  reminder_volunteer_path(@volunteer),
-                  method: :patch,
-                  class: "btn btn-primary casa-case-button",
-                  data_toggle: "tooltip",
-                  title: "#{t(".tooltip")}" %>
```

Comment:
let's not lose this tooltip by accident
if we want to remove it on purpose, please take out the text in en.yml also